internal/runners/ppm/convert.go

```diff
@@ -0,0 +1,191 @@
+package ppm
+
+import (
+	"github.com/ActiveState/cli/internal/analytics"
+	"github.com/ActiveState/cli/internal/constants"
+	"github.com/ActiveState/cli/internal/failures"
+	"github.com/ActiveState/cli/internal/locale"
+	"github.com/ActiveState/cli/internal/logging"
+	"github.com/ActiveState/cli/internal/output"
+	"github.com/ActiveState/cli/internal/textutils"
+	"github.com/skratchdot/open-golang/open"
+)
+
+// convertAnswerCreate is the answer that the user can choose if they accept to create a virtual environment.  It is re-used at several places.
+var convertAnswerCreate string = locale.Tl("ppm_convert_answer_create", "Create Virtual Runtime Environment")
+
+// analyticsEventFunc is used to send analytics event
+type analyticsEventFunc func(string, string, string)
+
+// surveySelectFunc displays a menu with options that the user can select
+type surveySelectFunc func(message string, choices []string, defaultResponse string) (string, *failures.Failure)
+
+const (
+	askedWhy          string = "asked why"
+	seenStateToolInfo        = "visited state tool info"
+	seenPlatformInfo         = "visited platform info"
+	notConvinced             = "still wants ppm"
```

Comment:
These don't seem to be used

Reply:
They should have been, maybe this was still on an older commit...